resources/resource/resource_helpers.go

```diff
@@ -55,6 +58,11 @@ func getParam(r Resource, key string, stringToLower bool) any {
 		return cast.ToFloat64(v)
 	case time.Time:
 		return val
+	case toml.LocalDate, toml.LocalDateTime:
+		if vt, ok := hreflect.AsTime(reflect.ValueOf(val), time.UTC); ok {
```

Comment:
Using reflect here isn't needed. You can just do `val.AsTime(time.UTC)`.

Reply:
I couldn't quite figure out how to do what you describe, but hopefully using `htime.ToTimeInDefaultLocationE(val, time.UTC)` is sufficient.